### DISCUSSION TITLE
how can run metasploit in termux

### DISCUSSION THREAD

**Discussion by inhdyaii** [rapid7/metasploit-framework#14779] [Q&A] [answered]
Iam cant open meta in termux can you pleas help me

**Comment by bcoles** [comment, marked as answer]
Termux is not officially supported. #11023

However, Metasploit reportedly does work with Termux.

Refer to the following for more information:

* https://wiki.termux.com/wiki/Metasploit_Framework
* termux/termux-packages/issues/715
